skills_review/templates/index.pug

```diff
@@ -7,4 +7,5 @@ block content
       form(method="POST")
         {{csrf_input}}
 
+        <input type="hidden" name="source" value="{{source|default('openai')}}">
```

Comment:
How does the user actually switch between openAI and taxonomy sources? Or is this a system setting we're using?

Reply:
Offline convo: URL matches switch between openai and taxonomy